keyboards/doio/kb38/config.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Katrina (@Daggette10)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#include "config_common.h"
+
+#define RGB_DI_PIN F6
+#define RGB_MATRIX_LED_COUNT 44
+
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200
+#define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_COLOR
+#define ENABLE_RGB_MATRIX_ALPHAS_MODS
+#define ENABLE_RGB_MATRIX_GRADIENT_UP_DOWN
+#define ENABLE_RGB_MATRIX_GRADIENT_LEFT_RIGHT
+#define ENABLE_RGB_MATRIX_BREATHING
+#define ENABLE_RGB_MATRIX_CYCLE_ALL
+#define ENABLE_RGB_MATRIX_RAINBOW_MOVING_CHEVRON
+#define RGB_MATRIX_DEFAULT_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+
+#define ENCODERS_PAD_A { C7, F7, D6 }
+#define ENCODERS_PAD_B { C6, E6, D7 }
+#define ENCODER_RESOLUTIONS { 4, 4, 2 }
```

Comment:
Is there a particular reason for the setting?

Edit:  Ah, different sized knobs